### DISCUSSION TITLE
Porting your package to use Microsoft reverse proxy

### DISCUSSION THREAD

**Discussion by radrad** [Burgyn/MMLib.SwaggerForOcelot#140] [Ideas]
Hi,
I already sent you an email about it. I see that a similar approach of embedding swagger configuration into reverseproxy configuration section and using remote micro services paths for matching purpose. 
can you please advice what to do to enable similar integration
https://github.com/microsoft/reverse-proxy/blob/master/samples/ReverseProxy.Config.Sample/appsettings.json

Thank you
Rad

**Comment by Burgyn** [comment]
Hi @radrad,
thank you for sharing your idea.

I quickly looked at the YARP project. Yes, you are right. The idea of my package can be used for it. But the implementation of Ocelot and YARP is very different. I can imagine that the core of this package would be extracted to separate package and a new package for YARP would be created on that basis. 

If you need it very much, or there will be a lot of people who need it, that so I can help with the implementation. But that's not my priority right now.
